src/modules/reserve-overview/TokenLinkDropdown.tsx

```diff
@@ -66,25 +68,40 @@ export const TokenLinkDropdown = ({ poolReserve, downToSM }: TokenLinkDropdownPr
             address: poolReserve?.underlyingAsset,
           })}
           target="_blank"
+          divider
         >
           <TokenIcon symbol={poolReserve.iconSymbol} sx={{ fontSize: '20px' }} />
           <Typography variant="subheader1" sx={{ ml: 3 }} noWrap data-cy={`assetName`}>
             {poolReserve.symbol}
           </Typography>
         </MenuItem>
 
+        <Box sx={{ px: 4, pt: 3, pb: 2 }}>
+          <Typography variant="secondary12" color="text.secondary">
+            <Trans>Aave aToken</Trans>
+          </Typography>
+        </Box>
+
         <MenuItem
           component="a"
           href={currentNetworkConfig.explorerLinkBuilder({
             address: poolReserve?.aTokenAddress,
           })}
           target="_blank"
+          divider={showDebtTokenHeader}
         >
           <TokenIcon symbol={poolReserve.iconSymbol} aToken={true} sx={{ fontSize: '20px' }} />
           <Typography variant="subheader1" sx={{ ml: 3 }} noWrap data-cy={`assetName`}>
             {'a' + poolReserve.symbol}
           </Typography>
         </MenuItem>
+        {showDebtTokenHeader && (
+          <Box sx={{ px: 4, pt: 3, pb: 2 }}>
+            <Typography variant="secondary12" color="text.secondary">
+              <Trans>Aave Debt Token</Trans>
```

Comment:
Only the first word is capitalized in the first two headers, this one should probably be updated to match

Reply:
good catch, fixed